testing/regress/ecl/httpcall_multiheader.ecl

```diff
@@ -1,3 +1,23 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//class=spray
+//class=roxieserviceaccess
```

Comment:
why has 'roxieserviceaccess' been added, does this query access roxie?

Reply:
It was a mistake. Removed (as well as the "//class=spray" based on this isn't perform any spray operation)